### PR TITLE
:book: adding investigation for Edge Multicluster

### LINKED ISSUE #2392: feature: KCP.io Investigation Document for Edge Multicluster
### Feature Description

We would like to represent the Edge MC work/sub-project in KCP.io as a KCP sub-project.

### Proposed Solution

adding an Edge MC investigation MD

### Alternative Solutions

_No response_

### Want to contribute?

- [X] I would like to work on this issue.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by clubanderson**
## Summary
add investigation document for Edge Multicluster

## Related issue(s)

Fixes #2392

### CODE REVIEW

File: docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+description: >
+  Investigation of edge multi-cluster
+---
+
+Kubernetes offers a state-based management platform to declaratively developing and operating workloads. Kube allows a developer to
+ create workloads that are node-agnostic and can be constrained to operate in places where certain conditions are met. Operators enjoy
+  gaurantee that workloads are rescheduled to work wherever constraints allow them to run. This approach has satisfied the large
+   majority of use cases in the cloud native community.
+
+There is a growing community of cloud native developers and operators that work for organizations that require more fine-grain control
```
Comment (pdettori) [inline docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md:14]: fine-grain -> fine-grained

File: docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+## Hierarchy and Scale are core considerations
+
+KCP has introduced the concept of transparent multi-cluster which can be used to expand the number of entry points for workloads and
+ operations by reducing the size of the control plane to a single binary. Synchronization between KCP virtual workspaces and physical
+  clusters has made it possible to deploy workloads centrally and then fan out to any defined subset of clusters. We would like to
+   leverage multiple levels of hierarchy (cascading multiple instances of KCP and Kubernetes cluster) to arrive at a distributed system
+    capable of reaching any targeted edge cluster with/without edge devices attached. Building out such a hierarchy will require a
+     scalable database to hold the objects referenced by each of their associated API servers. The default database for KCP and
+      Kubernetes is etcd. We will investigate alternatives to etcd distribution and replacement databases to increase scalability with
+       sacrificing performance and resilience without negatively impacting cost.
```
Comment (pdettori) [inline docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md:50]: " with sacrificing performance and resilience without negatively impacting cost." This sentence is not clear -  I  suggest to rephrase it as: "without sacrificing performance and resilience and considering factors which could negatively impact cost.

File: docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md (hunk 1 above)
Comment (clubanderson) [reply docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md:14]: updated to "fine-grained"

File: docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+title: "Edge multi-cluster"
+linkTitle: "Edge multi-cluster"
+weight: 1
+description: >
+  Investigation of edge multi-cluster
+---
+
+Kubernetes offers a state-based management platform to declaratively developing and operating workloads. Kube allows a developer to
+ create workloads that are node-agnostic and can be constrained to operate in places where certain conditions are met. Operators enjoy
+  gaurantee that workloads are rescheduled to work wherever constraints allow them to run. This approach has satisfied the large
```
[the remark this answers is not in the document]
Comment (clubanderson) [reply docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md:11]: updated to "guarantees" as suggested

**Pull request closed without merging**